public/app/features/alerting/unified/api/alertSilencesApi.ts

```diff
@@ -47,6 +51,14 @@ export const alertSilencesApi = alertingApi.injectEndpoints({
         data: payload,
       }),
       invalidatesTags: ['AlertmanagerSilences', 'AlertmanagerAlerts'],
+      onQueryStarted: async (arg, { queryFulfilled }) => {
+        try {
+          await queryFulfilled;
+          appEvents.emit(AppEvents.alertSuccess, ['Silence created']);
+        } catch (error) {
+          appEvents.emit(AppEvents.alertError, ['Could not create silence']);
```

Comment:
Should we track these errors in faro?

Reply:
As per Slack - are you ok if we don't track this? As this is just the same as the error at the API level, rather than a UI problem/bug that we've encountered